.github/workflows/ci.yml

```diff
@@ -20,13 +20,33 @@ jobs:
       - run: yarn --frozen-lockfile --network-timeout 1000000
       - run: yarn test
 
-  acceptance:
+  integration:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         node-version: [16.x]
         os: [ubuntu-latest, macos-latest]
     environment: AcceptanceTests
+    env:
+      RUN_ACCEPTANCE_TESTS: true
```

Comment:
Do we want a new env var here to maintain the new distinction between acceptance/integration?

Reply:
What I'd like to do, really, is get rid of this env var entirely. I can't remember why I didn't here... let me see what I can do.

Reply:
Ah, I remember why now. I didn't want to re-do a bunch of work from the orgs-v5 PR to make this pass. I'll leave it for now, though, and I can wait to merge this until that PR is in.